### DISCUSSION TITLE
Keda Scaledobject is not working with aws-cloudwatch with multiple dimensions

### DISCUSSION THREAD

**Discussion by phanisowjanyavutukuri** [kedacore/keda#4804] [Q&A / Need Help]
Hi,

I am using the scaledobject with multiple dimensions seperated by ; but still the scaledobject is in not ready state, but if i use the expression instead of the key values then it is working.

could you please help me

**Comment by JorTurFer** [comment]
Which is the error that you see in KEDA operator logs?
